.eslintrc.json

```diff
@@ -44,8 +57,8 @@
     }
   ],
   "rules": {
-    "id-length": "warn",
-    "no-magic-numbers": "warn",
+    "id-length": "off",
+    "no-magic-numbers": ["off", {"ignoreArrayIndexes": true}],
```

Comment:
Do we need this config here when the rule is `off`? (`{"ignoreArrayIndexes": true}`)

Reply:
its there in case we want to turn the rule on again. this setting doesn't hurt and we don't forget it then :D